test/variational/test_exact_state.py

```diff
@@ -89,6 +89,19 @@ def _f(x, y):
     jax.tree_multimap(_f, pars, pars2)
 
 
+@common.skipif_mpi
+def test_basic_methods(vstate):
+    key1, key2 = jax.random.split(nk.jax.PRNGKey())
+
+    s = vstate.hilbert.random_state(key1)
+    assert np.shape(vstate.log_value(s)) == ()
+
+    s = vstate.hilbert.random_state(key2, size=2)
+    assert np.shape(vstate.log_value(s)) == (2,)
+
+    _ = vstate.quantum_geometric_tensor()
```

Comment:
Can we check anything of the returned object?

Reply:
Maybe something like `hasattr(qgt, "solve")`? The QGT classes don't have a common base type.

Reply:
nk.optimizer.LinearOperator I think